include/boost/beast/core/impl/basic_stream.hpp

```diff
@@ -326,13 +346,6 @@ class transfer_op
 
                     async_perform(0, is_read{});
```

Comment:
@vinniefalco I believe the use of `async_perform(0, is_read{})` here has been to reduce the number of template instantiations. If that's the case, I can replace the use of `net::dispatch` with `async_perform(0, is_read{})` for the same reason.

Reply:
seems OK I guess...I think you know more about this than me now.